internal/policy/helpers_test.go

```diff
@@ -203,12 +203,12 @@ func createTestStateWithDelegatedPolicies(t *testing.T) *State {
 
 	// Create the second level of delegations
 	delegation1Metadata := InitializeTargetsMetadata()
-	delegation1Metadata, err = AddOrUpdateDelegation(delegation1Metadata, "3", []*tuf.Key{gpgKey}, []string{"file:1/subpath1/*"})
+	delegation1Metadata, err = AddOrUpdateDelegation(delegation1Metadata, "3", []*tuf.Key{gpgkey}, []string{"file:1/subpath1/*"})
```

Comment:
This diff is because of the confusion with `gpgKey`. If you make it so gpgKey has no diff (as I say above), this line also should have no diff.

Reply:
Fixed